src/assets/socis.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function SOCIS({ width, height, viewbox }) {
+  return (
+    <svg
```

Comment:
Why not include the vector as a separate .svg file so its easier to see where it is. I think most people would not expect to find this in a .jsx file

Reply:
logos were put here so they have have properties or something @SoorajModi had a reason that sounded reasonable.